.github/workflows/release.yml

```diff
@@ -0,0 +1,169 @@
+name: release
+
+on:
+  push:
+    tags:
+      - '*'
+
+defaults:
+  run:
+    shell: bash # Windows' default shell is PowerShell
+
+env:
+  CI_RELEASE: true
+
+jobs:
+  draft_release:
+    name: Create release
+    runs-on: ubuntu-latest
+    outputs:
+      upload_url: ${{ steps.create_release.outputs.upload_url }}
+
+    steps:
+      - name: Create Release
+        id: create_release
+        uses: actions/create-release@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          tag_name: ${{ github.ref }}
+          release_name: ${{ github.ref }}
+          body: ""
+          draft: false
+          prerelease: false
+          commitish: master
+
+  build_release:
```

Comment:
I don't think we should duplicate all of this functionality across both the release.yml workflow and the ci.yml workflow. Can we instead set up the ci.yml workflow so that if the build succeeds and we are building a tag, we move on to a release.yml workflow which still has access to the compiler we've just built so that it only needs to contain the logic for creating and uploading a release?

Reply:
I'm not sure. I imagine it can? 
It's been a while since I've read through GH Actions' documentation. Moreover, I'm still not familiar with how the releasing works. Lastly, I wasn't sure how to translate the `.travis.yml` file into a corresponding one for GH Actions, mainly because I don't make releases for PureScript.